README.md

```diff
@@ -90,6 +95,33 @@ const output = diff(firstDocument, secondDocument, {
 });
 ```
 
+## Output
```

Comment:
@aayushmau5 Not sure, maybe we can present the different possible values in a tabular form?

Reply:
@vinitshahdeo What do you mean by `tabular form` 🤔 ?